shell.nix

```diff
@@ -56,6 +56,14 @@ mkDerivation {
     export NIX_PATH=nixpkgs=${nixpkgs}
     export NIX_PATH_LOCKED=1
     echo   NIX_PATH set to $NIX_PATH >&2
+
+    alias io-csl-path-commit='echo -n "Enter cardano-sl nix store path: "; read csl_path; nix-store --query --references $(nix-store --query --deriver $csl_path) | grep cardano-sl- | grep -v sl-core\\\|sl-db\\\|sl-infra\\\|sl-lrc\\\|sl-ssc\\\|sl-godtossing\\\|sl-txp\\\|sl-update | cut -c56-62'
```

Comment:
This duplicates the new command `dump-commits`, we ought to have one code to show the versions/commits used.